cmd/thv-operator/api/v1alpha1/mcpregistry_types.go

```diff
@@ -16,6 +16,9 @@ const (
 
 	// RegistrySourceTypeAPI is the type for registry data fetched from API endpoints
 	RegistrySourceTypeAPI = "api"
+
+	// RegistrySourceTypePVC is the type for registry data stored in PersistentVolumeClaims
+	RegistrySourceTypePVC = "pvc"
 )
```

Comment:
This constant and also the others above can be safely removed IMO (missed to drop them in another cleanup task): can you pls remove all the `const` block?